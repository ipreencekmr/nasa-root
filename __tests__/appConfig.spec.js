import AppConfig from '../src/config/appConfig';

describe('appConfig', () => {
  it('shoule be valid json object', () => {
    expect(AppConfig).toBeInstanceOf(Object);
    expect(Object.keys(AppConfig)).toHaveLength(3);
  });

  it('should contain csp', () => {
    expect(AppConfig.csp).toBeDefined();
    expect(AppConfig.provideStateConfig).toBeDefined();
    expect(AppConfig.pwa).toBeDefined();
    expect(typeof AppConfig.csp).toBe('string');
    expect(typeof AppConfig.provideStateConfig).toBe('object');
    expect(typeof AppConfig.pwa).toBe('object');
  });
});
