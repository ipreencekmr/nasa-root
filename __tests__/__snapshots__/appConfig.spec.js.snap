// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`appConfig csp should be a valid csp string 1`] = `"report-uri; default-src 'self'; script-src 'self'; img-src 'self'; style-src 'self' 'unsafe-inline'; connect-src 'self'"`;
